camlibs/ptp2/config.c

```diff
@@ -2460,6 +2460,89 @@ _put_Olympus_ISO(CONFIG_PUT_ARGS)
 	}
 	return GP_ERROR;
 }
+//Modif Berylius: Add OMD Drive Mode Widget
+/* Work in progress
```

Comment:
What about this part?

Reply:
I started it but i need to sniff the USB trafic to finish the function. 
It has no impact on the Bulb mode function (main reason for the pull request)

Reply:
Therefore I'd remove it for the time being and re-introduce it in another PR once you'll get things working.